insync/PKGBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: Nate Levesque <public@thenaterhood.com>
+# Contributor: Vlad M. <vlad@archlinux.net>
+# Contributor: Zhengyu Xu <xzy3186@gmail.com>
+
+pkgname=insync
+pkgver=1.5.5
+_pkgver=37367
+_dist=artful
+pkgrel=1
+pkgdesc="An unofficial Google Drive client that runs on Linux, with support for various desktops"
+url="https://www.insynchq.com/downloads"
+license=('custom:insync')
```

Comment:
Does its license allow binary redistribution?

Reply:
I don't find license information on its home page. 

well, this software itself distribute a `.deb` file, I think it's not a big problem?  Just like enpass-bin that already in archlinuxcn repo.